### PR TITLE
Update jquery.djax.js

### PR DESCRIPTION
**Pull request by WTVader**
Compatibility with jQuery 1.9

### CODE REVIEW

**Comment by beezee** [comment]
Looks simple enough - is this all that's required? Does it maintain backwards compatibility?

**Comment by WTVader** [comment]
Yes! I've tried it also with jQuery 1.8.3 and it worked well.

**Comment by beezee** [comment]
Thanks- tested on my live demo (http://brianzeligson.com/djax) and it breaks (no $.parseHTML available in jQuery 1.4.4)

I realize that's a pretty old version, but the fix for me was as simple as 

```
var result = (typeof $.parseHTML === 'function')
                ? $($.parseHTML(response)) : '"'+response+'"';
```

This has the advantage of using $.parseHTML when it's available, and falling back on the old method when it's not, and should keep things working in all versions. If you'd like to make this adjustment I'll be happy to merge the pull.

**Comment by WTVader** [comment]
Sounds good. Thanks for that!
There is another issue I came across recently: djax doesn't usually work for (wordpress) search form submits. Do you have any idea how to modify djax so that it works also for search pages?

**Comment by beezee** [comment]
Likely because djax is bound to links, as opposed to this context being a form submit.

The easiest solution is probably to update a hidden link based on the form input and trigger a click instead of allowing default form submission, something like this:

```
<form id="search" action="/">
    Search: <input id="search-term" type="text" />
</form>
<a id="search-submit" style="display:none;"></a>
<script type="text/javascript>
    function($, exports, undefined){
        $(document).ready(function() {
            $('#search').submit(function(e){
                e.preventDefault();
                $('#search-submit').attr('href', '/?s='+encodeURIComponent($('#search-term').val()))
                    .click();
            });
        });
    }(jQuery, window));
</script>
```

Keep me posted on your pull request and I'll merge once it's adjusted.

**Comment by kyleconrad** [comment]
For those wondering how to do search, here's how I got it working all nice and pretty inside djax.

HTML:

```
<form role="search" class="search" method="get" id="searchform" action="<?php echo home_url( '/' ); ?>" autocomplete="off">
    <input type="text" value="" name="s" id="search" class="searchbar" />
</form>
<a href="#" id="search-submit"/></a>
```

JQUERY:

``` javascript
$('#searchform').submit(function(e){
    event.preventDefault();
    $('#search-submit').attr('href', '<?php bloginfo('url');?>/?s='+encodeURIComponent($('#search').val())).trigger('click');
});
```
